src/L.Realtime.js

```diff
@@ -10,11 +10,12 @@ L.Realtime = L.Layer.extend({
         updateFeature: function(f, oldLayer) {
             if (!oldLayer) { return; }
 
-            if (f.geometry.type === 'Point') {
+            if (oldLayer.setLatLng) {
                 var c = f.geometry.coordinates;
                 oldLayer.setLatLng([c[1], c[0]]);
                 return oldLayer;
             }
+            return oldLayer;
```

Comment:
The current situation is, that leaflet loses references to old polygons. These objects will keep initial properties like popups just for the first loading. The reload is clearing all these settings. Other ideas?